internal/sql/repository/AppListingRepository.go

```diff
@@ -116,31 +116,39 @@ func (impl AppListingRepositoryImpl) FetchAppsByEnvironment(appListingFilter hel
 
 	latestDeploymentStatusMap := map[string]*bean.AppEnvironmentContainer{}
 	for _, item := range appEnvContainer {
-
-		key := strconv.Itoa(item.AppId) + "_" + strconv.Itoa(item.EnvironmentId)
-		if _, ok := latestDeploymentStatusMap[key]; ok {
-			continue
+		skip := false
+		if item.EnvironmentId > 0 && item.PipelineId > 0 && item.Active == false {
```

Comment:
what if EnvironmentId == 0 && PipelineId == 0 && Active == false ?

Reply:
items should add in app listing in this case.